draft-chfl-intarea-grim.xml

```diff
@@ -126,16 +160,138 @@ TBA
     <section anchor="challenges">
 <name>Problem Statement</name>
 <t>
-TBA
+Even when a link supports a large MTU, two challenges emerge:
+</t>
+<ul>
+<li>
+<t>
+All nodes on the link must utilize the same MTU value (e.g., advertised via the MTU RA option or derived from explicit or implicit interface configuration). An administrator cannot configure nodes with a larger MTU until all nodes on the link are updated.
+</t>
+</li>
+<li>
+<t>
+When a node configures a large MTU on its interface (either through an MTU RA option or other means), it may employ that MTU as a default starting value for all communications, both on-link and off-link. While all links within an administrative domain may support large MTUs, it is improbable that all Internet paths do so. Given the inherent unreliability of PMTUD, the node will inevitably encounter MTU issues when communicating with Internet destinations.
```

Comment:
I' m not sure the "Given the unreliability..." part reads quite correctly. The problem as I see it is (even with a perfect MTU discovery method), "a node will inevitably encounter a PMTU size that varies depending on the Internet destinations".

Reply:
But....isn't MTU is stored in the destination cache? So per-destination MTU is OK, the problem (at least in my experience) is that we keep seeing backholing caused by MTU not being detected correctly...

Anyway - suggestions for the text would be appreciated ;)